bbr/bbrLib.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+#include <chrono>
+#include <algorithm>
+
+
+enum class stage{
+    BBR_STARTUP,
+    BBR_DRAIN,
+    BBR_NORMAL, 
+    BBR_PROBE_BW,
+    BBR_CHECK_DOWN_SPEED,
+    BBR_PROBE_BW_CHECK,
+    BBR_PROBE_RTT
```

Comment:
BBR states are `STARTUP`, `DRAIN`, `PROBE_BW` and `PROBE_RTT`.
In BBRv2 `PROBE_BW` has following phases: `Up`, `Down`, `Cruise` and `Refill`.

So to me it's not clear what are the `BBR_NORMAL`, `BBR_CHECK_DOWN_SPEED` and `BBR_PROBE_BW_CHECK`